eng/pipelines/libraries/enterprise/linux.yml

```diff
@@ -0,0 +1,97 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+    - release/*.*
+  paths:
+    exclude:
+    - docs/*
+    - CODE-OF-CONDUCT.md
+    - CONTRIBUTING.md
+    - LICENSE.TXT
+    - PATENTS.TXT
+    - README.md
+    - SECURITY.md
+    - THIRD-PARTY-NOTICES.TXT
+    - src/installer/*
+    - src/coreclr/*
+    - eng/pipelines/coreclr/*
+    - eng/pipelines/installer/*
+    - eng/pipelines/common/*
+
+pr:
+  branches:
+    include:
+    - master
```

Comment:
Do we want these tests to trigger automatically on every PR ?

Reply:
My goal is 'yes'. I would like these tests to run for every PR in the same way we run innerloop tests in general. I consider these new 'enterprise scenario' tests to be innerloop.

Reply:
Since this is a new pipeline, I'm going to initially restrict the 'auto-run on PR' feature to just changes in certain high-priority network-related filepaths in the repo. As we gain experience from this feature, we'll adjust over time.